.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,23 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
```

Comment:
Just to be clear, this doesn't run any of the actual tests, right?

Reply:
You are right.
```suggestion
# This workflow will run lint
```